proto/regen/ecocredit/v1/tx.proto

```diff
@@ -118,6 +118,29 @@ service Msg {
   // credit types to the network.
   // Since Revision 1
   rpc AddCreditType(MsgAddCreditType) returns (MsgAddCreditTypeResponse);
+
+  // AddClassCreator is a governance method that allows the addition of new
+  // address to the class creation allowlist.
+  // Since Revision 1
+  rpc AddClassCreator(MsgAddClassCreator)
+      returns (MsgAddClassCreatorResponse);
+
+  // RemoveClassCreator is a governance method that removes
+  // address from the creation allowlist.
+  // Since Revision 1
+  rpc RemoveClassCreator(MsgRemoveClassCreator)
+      returns (MsgRemoveClassCreatorResponse);
+
+  // ToggleCreditClassAllowlist is a governance method that toggles the network
+  // allowlist to on or off. when on, the class creator allowlist is used to
+  // enforce which addresses may create classes. when off, any address may
+  // create classes. Since Revision 1
+  rpc ToggleCreditClassAllowlist(MsgToggleCreditClassAllowlist)
+      returns (MsgToggleCreditClassAllowlistResponse);
```

Comment:
This should probably be `ToggleClassCreatorAllowlist` but we may also want to update based on #1387 so this is ok for now to keep things moving and I've added a comment in the issue https://github.com/regen-network/regen-ledger/issues/1387#issuecomment-1224292001.